### PR TITLE
Dijkstra’s Shortest Path Algorithm

### PR DESCRIPTION
**Pull request by RobbiNespu**
Source taken from https://algocoding.wordpress.com/2015/03/28/dijkstras-algorithm-part-4a-python-implementation/

### CODE REVIEW

**Comment by aneels3** [comment]
hello @RobbiNespu please change the PR name implementation to the original issue name.

File: Dijktra.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Dijktra's shortest path algorithm. Prints the path from source to target.
+ 
+
```
Comment (aneels3) [inline Dijktra.py]: please delete the extra spaces in your code.

Comment (RobbiNespu) [reply Dijktra.py]: Hi @aneels3 , I do the changes

Comment (aneels3) [reply Dijktra.py]: great!
It will be great if you add some algorithm problem by creating an issue.

Comment (aneels3) [reply Dijktra.py]: Nice @RobbiNespu 
It will be great if you add some algorithm problem by creating an issue.

**Pull request merged**